privacy.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: markdown
+title: Privacy Policy
+description: This Privacy Policy describes how we collect, use, and safeguard the information you give us.
+---
+
+*Effective date: 30 September 2020*
+
+This Privacy Policy describes how we collect, use, and safeguard the information you give us. In this agreement, website refers to the Scratch Addons website (<https://scratchaddons.com/>). Extension refers to our browser extension, which you can download from <https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco> or <https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/.>
+
+When we talk about the service, we mean any service provided by us, such as the website or our services used in addons. The terms &quot;we,&quot; &quot;us,&quot; and &quot;our&quot; refer to the leaders of the Scratch Addons open source project (<https://github.com/ScratchAddons>). When you use our services, you agree to this privacy policy.
+
+## Information we collect
+
+We collect a few different kinds of information about you, like what part of the world you are in, what pages you come to our website from, and what device and software you use. We also collect information about how you use our website. We might collect your Scratch username.
```

Comment:
> We might collect your Scratch username.

Any specific reason for this to not be specific enough? We don't collect usernames unless explicitly given by users, in the feedback page.

Reply:
cc @easrng